src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -204,7 +204,39 @@
     </Copy>
   </Target>
 
-  <!-- archive the test binaries along with some supporting files -->
+  <!-- Workaround for VS execution:  This will form the same list and copy the same files as
+       copied via RunTests script so VS can work when the test dir is initially clean. 
+       -->
+  <Target Name="CopyRunnerScriptFiles" Condition=" '$(BuildingInsideVisualStudio)'=='true' "
+        AfterTargets="CopySupplementalTestData">
+    <Message Text ="Copying over test dependencies since we're running inside Visual studio." />
```

Comment:
I would remove this message, I'm not even sure it will actually show up anywhere.